packages/datadog-plugin-gcloud-pubsub/test/index.spec.js

```diff
@@ -0,0 +1,153 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const { expectSomeSpan, withDefaults } = require('../../dd-trace/test/plugins/helpers')
+const plugin = require('../src')
+
+const BASE_PROJECT_ID = `test-project-${bigRandom()}-`
+let projectCounter = 0
+const BASE_TOPIC = `test-topic-${bigRandom()}-`
+let topicCounter = 0
+
+wrapIt()
+
+describe('Plugin', () => {
+  let tracer
+  let gpubsub
+
+  describe('gcloud-pubsub', () => {
+    before(() => {
+      process.env.PUBSUB_EMULATOR_HOST = 'localhost:8042'
+    })
+    after(() => {
+      delete process.env.PUBSUB_EMULATOR_HOST
+    })
+    afterEach(() => {
+      agent.close()
+      agent.wipe()
+    })
+    withVersions(plugin, '@google-cloud/pubsub', version => {
+      describe('without configuration', () => {
+        beforeEach(() => {
+          tracer = require('../../dd-trace')
+          agent.load(plugin, '@google-cloud/pubsub')
```

Comment:
This should be the plugin name and not the module name. I'm actually not even sure why it works right now 😐